cansat-core/src/lib.rs

```diff
@@ -3,10 +3,36 @@
 #![no_std]
 
 use libm::powf;
+pub struct Pressure {
+    value: f32,
+}
+
+impl Pressure {
+    pub const fn new(_value: f32) -> Pressure {
+        Pressure {value: (_value) }
+    }
+
+    pub const fn from_pascals(value: f32) -> Pressure {
```

Comment:
In theory this `const` is appropriate here and it may be useful, but we don't use const evaluated functions anywhere in our code so I'd suggest removing it

```suggestion
    pub fn from_pascals(value: f32) -> Pressure {
```